zerver/lib/onboarding.py

```diff
@@ -214,7 +219,15 @@ def send_welcome_bot_response(send_request: SendMessageRequest) -> None:
     welcome_bot = get_system_bot(settings.WELCOME_BOT, send_request.message.sender.realm_id)
     human_response_lower = send_request.message.content.lower()
     content = select_welcome_bot_response(human_response_lower)
-    internal_send_private_message(welcome_bot, send_request.message.sender, content)
+
+    # Note: Welcome bot doesn't trigger email/push notifications,
+    # as this is intended to be seen contextually in the application.
```

Comment:
Let's move this comment inside the call, just before the new parameter -- that comment placement is more readable.